test/runner.rb

```diff
@@ -1,11 +1,20 @@
 # frozen_string_literal: true
 
+require 'rbconfig'
+
 # Should be done in rubygems test files?
 ENV["GEM_SKIP"] = ENV["GEM_HOME"] = ENV["GEM_PATH"] = "".freeze
 
-# Get bundled gems on load path
-Dir.glob("#{__dir__}/../gems/*/*.gemspec")
-  .reject {|f| f =~ /minitest|test-unit|power_assert/ }
-  .map {|f| $LOAD_PATH.unshift File.join(File.dirname(f), "lib") }
+# Get bundled gems on load path, try install dir first
+gem_dir = "#{RbConfig::CONFIG['rubylibdir'].sub('/ruby/', '/ruby/gems/')}/gems"
+unless Dir.exist? gem_dir
+  gem_dir = "#{__dir__}/../gems"
+end
+
+# we need to pick up gems without a gemspec, so pick folders with a lib sub-folder
+# the 'tz' gems for Windows do not have gemspec files
+Dir.glob("#{gem_dir}/*/lib")
+  .reject {|f| Dir.exist?(f) and f =~ /minitest|test-unit|power_assert/ }
+  .map {|f| $LOAD_PATH.unshift f }
```

Comment:
I believe you could use ``delete_if`` and ``each`` instead of ``reject`` and ``map``, maybe even
``$LOAD_PATH.concat(Dir.glob("#{gem_dir}/*/lib/").delete_if {|f|  f =~ /minitest|test-unit|power_assert/ }.reverse!)``.
I only added ``reverse!`` to obtain the same ordering, not sure if it is required.